alphadia/outputtransform.py

```diff
@@ -866,6 +874,14 @@ def _build_run_stat_df(
         if "mobility_fwhm" in channel_df.columns:
             base_dict["fwhm_mobility"] = np.mean(channel_df["mobility_fwhm"])
 
+        with open(optimization_manager_path, "rb") as f:
+            optimization_manager = pickle.load(f)
+
+        base_dict["ms2_error"] = optimization_manager.ms2_error
+        base_dict["ms1_error"] = optimization_manager.ms1_error
+        base_dict["rt_error"] = optimization_manager.rt_error
+        base_dict["mobility_error"] = optimization_manager.mobility_error
+
         out_df.append(base_dict)
```

Comment:
`out_df` is a bit misleading as it's not a dataframe .. maybe rename to `channel_data`?

Reply:
That is true, but it's consistent throughout the file so I'm inclined not to change it just for this, and I'm reluctant to change it throughout on this pull request.